pkg/plugins/policies/xds/testutil.go

```diff
@@ -18,3 +20,12 @@ func ResourceArrayShouldEqual(resources core_xds.ResourceList, expected []string
 	}
 	Expect(len(resources)).To(Equal(len(expected)))
 }
+
+func ParseDuration(duration string) *k8s.Duration {
```

Comment:
this is already implemented in: https://github.com/kumahq/kuma/blob/52c992533bb8ab5f565e56f1b7a9af0a96e361a7/pkg/test/api_types.go#L17